includes/Core/Admin/Standalone.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Admin\Standalone
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Admin;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Assets\Stylesheet;
+
+/**
+ * Class managing standlone mode.
+ *
+ * @since 1.0.0
+ * @access private
+ * @ignore
+ */
+final class Standalone {
+
+	/**
+	 * Plugin context.
+	 *
+	 * @since NEXT
+	 *
+	 * @var Context
+	 */
+	private $context;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since NEXT
+	 *
+	 * @param Context $context Plugin context.
+	 */
+	public function __construct( Context $context ) {
+
+		$this->context = $context;
+
+		$this->standalone_mode();
+
+	}
+
+	/**
+	 * Standalone mode
+	 *
+	 * @since NEXT
+	 */
+	public function standalone_mode() {
+
+		if ( ! $this->is_standalone() ) {
+
+			return;
+
+		}
+
+		add_filter( 'admin_body_class', array( $this, 'admin_body_classes' ) );
+
+		remove_action( 'in_admin_header', 'wp_admin_bar_render', 0 );
+
+		add_filter( 'admin_footer_text', '__return_empty_string', PHP_INT_MAX );
+		add_filter( 'update_footer', '__return_empty_string', PHP_INT_MAX );
+
+		$styles = new Stylesheet(
+			'googlesitekit-standalone-css',
+			array(
+				'src' => $this->context->url( 'dist/assets/' ) . 'css/standalone.css',
+			)
+		);
+
+		$styles->register();
+		$styles->enqueue();
```

Comment:
Rather than introducing a stylesheet here for these few lines of CSS, I'd prefer if we printed the CSS inline in an `admin_head` callback.

_If_ the decision is to use a CSS file, it should instead be handled by the `Assets` class, and only be enqueued from here, within the `admin_enqueue_scripts` hook. But I think it's not worth doing that for these few lines of CSS, let's print it inline.

Reply:
Yeah, I think that inlining those styles make the most sense.